index.rst

```diff
@@ -288,6 +288,13 @@ Welcome to PyTorch Tutorials
    :link: advanced/static_quantization_tutorial.html
    :tags: Image/Video,Quantization,Model-Optimization
 
+.. customcarditem::
+   :header: PyTorch Mobile Performance Recipes
+   :card_description: List of recipes for performance optimizations for using PyTorch on Mobile.
+   :image: _static/img/thumbnails/cropped/experimental-Dynamic-Quantization-on-an-LSTM-Word-Language-Model.png
+   :link: recipes/recipes/mobile_perf.html
```

Comment:
I think the CI is failing because we need add the ellipses because the recipes are nested one level deeper -  `.. /recipes/recipes/mobile_perf.html`